cristin-import/src/test/resources/features/BookAnthologyRules.feature

```diff
@@ -0,0 +1,14 @@
+Feature: Mapping of Book Anthology
+
+  Background:
+    Given a valid Cristin Result with secondary category "ANTOLOGI"
+
+
+  Scenario: map returns NVA Resource with Reference having a PublicationInstance of type
```

Comment:
I prefer that the scenario title starts with a capital letter, though there is no good reason for this except presentation

Reply:
will fix this one later.